ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Deployment/DeploymentPageVulnerabilities.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import {
+    Divider,
+    Flex,
+    PageSection,
+    Pagination,
+    pluralize,
```

Comment:
oh what i didn't even realize PF had its own pluralize 😮

Reply:
I know! The signature is slightly different, and I think we use a mix of both throughout, but I figured going with the PF one might let us drop a dependency in the future.